api/repositories/droplet_repository_test.go

```diff
@@ -373,7 +373,7 @@ var _ = Describe("DropletRepository", func() {
 				Expect(dropletRecords).To(HaveLen(2))
 			})
 
-			Describe("filtering", func() {
+			Describe("filtering by package guid", func() {
```

Comment:
No need to test filtering for every filtering param, see explanation below. Generally one test that filters by just one param is enough as a smoke test that proves that we have wired all the pieces together correctly